app/views/admin/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>Admin Dashboard</h1>
+
+<%= link_to 'Admin Merchants Index', admin_merchants_path %>
+<%= link_to 'Admin Invoices Index', admin_invoices_path %>
+
+<div id="successful_transactions">
+  <h3>Top 5 Customers with largest number of successful transactions:</h3>
+  <ol>
+    <% Customer.top_5_successful_transactions.each do |customer| %>
```

Comment:
Interesting to see the use of class methods here.  I've never seen this before.  That's cool.